pyp2spec/template.spec

```diff
@@ -12,17 +12,18 @@ Source:         {{source}}
 BuildArch:      noarch
 {%- endif %}
 BuildRequires:  python3-devel
-
+{% if additional_build_requires -%}
+# The package is probably archful, adding a dependency on a compiler
+{% for br in additional_build_requires -%}
+BuildRequires:  {{br}}
+{% endfor -%}
+{% endif %}
 
 # Fill in the actual package description to submit package to Fedora
 %global _description %{expand:
 {{description}}}
 
-{% if manual_build_requires -%}
-{% for br in manual_build_requires -%}
-BuildRequires:  {{br}}
-{% endfor -%}
-{% endif %}
+
```

Comment:
Nitpick: I'd consider removing this new line. It makes the specfile unnecessarily different from https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_example_spec_file and the two empty lines between %_description definition and actual %description look weird, especially considering there is only one empty line between %description and %package.